spec/integration/api/sys/mount_spec.rb

```diff
@@ -12,6 +12,15 @@ module Vault
         expect(sys.type).to eq("system")
         expect(sys.description).to include("system endpoints")
       end
+
+      it "shows options for mounts" do
+        next unless has_options_for_mount?
```

Comment:
could use skip so it's marked as skipped
```suggestion
        skip unless has_options_for_mount?
```